src/App.tsx

```diff
@@ -1,35 +1,29 @@
 import { Navigate, Route, Routes } from "react-router";
 import "./App.css";
-import Header from "./components/header/header";
 import Login from "./pages/login/Login";
 import Home from "./pages/home/Home";
 import Plans from "./pages/plans/Plans";
 import Students from "./pages/students/Students";
 import { PrivateRoute } from "./routes/PrivateRoutes";
 import StudentDetail from "./pages/student_detail/StudentDetail";
 import EditStudent from "./pages/edit_student/EditStudent";
-import useAuthentication from "./hooks/useAuthentication";
+import { MisAlumnos, NuevoAlumno, IniciarSesion, MisPlanes, DetalleAlumno, EditarAlumno } from "./routes/RoutesUtils";
+
 
 function App() {
-  const { isAuthenticated } = useAuthentication();
   return (
     <>
-      { isAuthenticated ? <Header /> : null }
-      <main className="content">
-        <Routes>
-          <Route path="/login" element={<Login />} />
-
-          <Route element={<PrivateRoute />}>
-            <Route path="/home" element={<Home />} />
-            <Route path="/cuotas" element={<Plans />} />
-            <Route path="/nuevo-alumno" element={<Students />} />
-            <Route path="/detalle-alumno" element={<StudentDetail />} />
-            <Route path="/editar-alumno" element={<EditStudent />} />
-          </Route>
-
-          <Route path="/" element={<Navigate to="/login" />} />
-        </Routes>
-      </main>
+      <Routes>
+        <Route path={IniciarSesion} element={<Login />} />
+        <Route element={<PrivateRoute />}>
+          <Route path={MisAlumnos} element={<Home />} />
+          <Route path={MisPlanes} element={<Plans />} />
+          <Route path={NuevoAlumno} element={<Students />} />
+          <Route path={DetalleAlumno} element={<StudentDetail />} />
+          <Route path={EditarAlumno} element={<EditStudent />} />
+        </Route>
+        <Route path="/" element={<Navigate to="/login" />} />
```

Comment:
Acá te faltó reemplazar en el Navigate


```suggestion
        <Route path="/" element={<Navigate to={IniciarSesion} />} />
```